models/_sector/nft/trades/MIGRATION.md

```diff
@@ -0,0 +1,156 @@
+
+#### Base marketplace models
+
+Base table schema (to be implemented by marketplace models):
+
+| Column Name              | Descriptions                                                               |
+|--------------------------|----------------------------------------------------------------------------|
+| **project**              | The name of the project/marketplace executing the trade                    |
+| **project_version**      | The version of the project/marketplace executing the trade                 |
+| **block_number**         | The number of the block in which the NFT trade occurred                    |
+| **tx_hash**              | The hash of the transaction in which the NFT trade occurred                |
+| **sub_tx_trade_id**      | A unique identifier of the NFT trade within one transaction                |
+| trade_category           | The category of the NFT trade (buy, sell, other)                           |
+| trade_type               | The type of NFT trade (primary/secondary)                                  |
+| buyer                    | The address of the buyer                                                   |
+| seller                   | The address of the seller                                                  |
+| nft_contract_address     | The NFT contract address                                                   |
+| nft_token_id             | The ID of the NFT token                                                    |
+| nft_amount               | The amount of NFTs (only >1 for ERC1155)                                   |
+| price_raw                | The raw full amount (ERC20) paid for the NFT, not adjusted for decimals    |
+| currency_contract        | The address of the contract for the currency used in the NFT trade         |
+| project_contract_address | The address of the contract for the project associated with the NFT trade  |
+| platform_fee_amount_raw  | The raw amount (ERC20) paid in marketplace fees, not adjusted for decimals |
+| platform_fee_address     | The address to which the platform fee was sent to (if applicable)          |
+| royalty_fee_amount_raw   | The raw amount (ERC20) paid in royalties, not adjusted for decimals        |
+| royalty_fee_address      | The address to which the royalty fee was sent to (if applicable)           |
+
+Enrichment of these base tables is done in 1 model for each chain.
+The enrichment logic can be found in `macros/models/sector/nft` and includes:
+
+1. adding transaction information
+2. adding NFT token information
+3. adding ERC20 token information + handle ERC20 decimals
+4. handle USD columns
+5. adding aggregator columns
+6. fixing buyer or seller for aggregator txs
+7. calculating platform and royalty rates
+
+
+
+
+### Migration
+
+#### User changelog
+
+The following changes affect the users:
+
+##### models:
+- [deprecated] `nft.events`, use `nft.trades` instead
+- [deprecated] `nft.fees`, use `nft.trades` instead
+
+
+##### columns:
```

Comment:
similar idea here to raise awareness on column changes
cc @augustog @0xBoxer @andrewhong5297

Reply:
https://dune.height.app/T-24057?task=T-27752